README.md

```diff
@@ -217,15 +216,15 @@ You will see the triangle with `!` under `Fairness` -> `Sex`. This indicates tha
 
 *Explanations show the most significant factors when determining an outcome. Classification models also include advanced explanations. Advanced explanations are not available for regression, image, and unstructured text models.*
 
-* Click on the info icon next to `Minimum changes for No Risk outcome` and look at the feature values:
+* Click on the info icon next to `Minimum changes for another outcome` and look at the feature values:
 
 *Pertinent Negative
-If the feature values were set to these values, the prediction would change. This is the minimum set of changes in feature values to generate a different prediction. Each feature value is changed so that it moves towards its median value in the training data.*
+Pertinent Negatives (PN) are feature values obtained by changing the value of each feature away from its median such that the model prediction changes. If the feeature attributes were set to these values, the prediction would change. This is the minimum set of changes in feature values to generate a different predictioon.*
 
 * Click on the info icon next to `Maximum changes allowed for the same outcome` and look at the feature values:
 
 *Pertinent Positive
-The prediction will not change even if the feature values are set to these values. This is the maximum change allowed while maintaining the existing prediction. Each feature value is changed so that it moves towards its median value in the training data.*
+Pertinent Postives (PP) are feature values obtained by changing the values of each feature towards its median such that thee model preediction remains the same. The prediction does not chang, even if the feature attributes are set to these values. This is the maximum possible change whille maintaining the existing prediction. Each feature value changes so that it moves towards it median value in the training data.*
```

Comment:
typos "thee", "preediction", "chang", "whille" and "it median value" near the end should be "its median value"

Reply:
fixed.